src/game/behaviors/goomba.inc.c

```diff
@@ -268,14 +247,14 @@ static void goomba_act_jump(void) {
 
 #if defined(FLOOMBAS) && defined(HD_INTRO_TEXTURES)
 static void floomba_act_startup(void) {
-    if (o->oBehParams & 0x00008000) { //! TODO: Flag name
+    if (GET_BPARAM3(o->oBehParams) & GOOMBA_BP3_IS_FLOOMBA) {
```

Comment:
This flag determines whether the Floomba animation should effectively be mirrored (for the sake of the start screen). Might be worth renaming `GOOMBA_BP3_IS_FLOOMBA` to something like `FLOOMBA_MIRRORED_ANIMATION` or something along those lines.